db/sql/10_thurstoncty.sql

```diff
@@ -8,4 +8,4 @@ CREATE TABLE IF NOT EXISTS thurston_officers (
 );
 
 COPY thurston_officers (title,last_name,first_name,call_sign,call_sign_2)
-FROM '/seed/ThurstonCounty-WA-Sheriffs-Office_05-01-21.csv' DELIMITER ',' CSV HEADER;
+FROM PROGRAM 'curl "https://techblocsea.sfo3.digitaloceanspaces.com/spd-lookup/ThurstonCounty-WA-Sheriffs-Office_05-01-21.csv"' DELIMITER ',' CSV HEADER;

```

Comment:
I wonder if these URLs could be templated...maybe then we could have these in minio locally for testing. What do you think?

Reply:
Oh that's a good idea, I bet we can. I'll look into it.

Reply:
Took quite a bit longer than I meant it to, but I've gotten this working. Will expand more on thoughts in another comment.